app/build.gradle

```diff
@@ -39,6 +39,7 @@ android {
         applicationId 'com.xmartlabs.template'
         minSdkVersion 17
         targetSdkVersion 24
+        testInstrumentationRunner 'com.franklin.app.TestRunner'
```

Comment:
Change to correct packageId. Maybe use it parametric in all places you can, so changing it once is enough.